pkgs/applications/science/chemistry/pymol/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, python27Packages, glew, freeglut, libpng, libxml2, tk, freetype }:
+
+let
+  version = "1.8.4.0";
+in
+stdenv.mkDerivation {
```

Comment:
There is also `buildPythonApplication` in `pythonPackages` which should automate all the phases you have overriden. Have you tried that?